api/provider/aws/logging.go

```diff
@@ -16,6 +16,73 @@ const (
 	MAX_DESCRIBE_STREAMS_COUNT = 1000
 )
 
+func GetLogsFromCloudTrail(
+	cloudWatchLogsAPI cloudwatchlogsiface.CloudWatchLogsAPI,
+	logGroupName string,
+	tail int,
+	start,
+	end time.Time,
+	filterPattern string,
+) ([]models.LogFile, error) {
+	logStreams, err := describeLogStreams(cloudWatchLogsAPI, logGroupName)
+	if err != nil {
+		return nil, err
+	}
+
+	logFiles := []models.LogFile{}
+	for _, logStream := range logStreams {
+		if strings.Contains(aws.StringValue(logStream.LogStreamName), "_CloudTrail_") {
```

Comment:
[nit] Instead of nesting a lot of code in a if block, I think it would be better to have a continue statement and the rest outside the if block. For example:

```go
for _, logStream := range logStreams [
    if !strings.Contains(...) {
        continue
    }
    
    // write the rest of your code here
}
```